src/docs/releases/1.9.0.md

```diff
@@ -152,3 +152,22 @@ public class AdminMenu : INavigationProvider
     }
 }
 ```
+
+### Admin Routes
+
+The `[Admin]` attribute now has optional parameters for a custom route template and route name. It works just like the `[Route(template, name)]` attribute, except it prepends the admin prefix. You can apply it to the controller or the action, if both are specified then the action's template takes precedence. The route name can contain `{area}`, `{controller}` and `{action}` which are substituted during mapping so the names can be unique for each action. This means you don't have to define these admin routes in your module's `Startup` class. Take a look at this example:
+
+```csharp
+[Admin("Person/{action}/{id?}", "Person{action}")]
+public class PersonController : Controller
+{
+    [Admin("Person", "Person")]
+    public IActionResult Index() { ... }
+
+    public IActionResult Create() { ... }
+
+    public IActionResult Edit(string id) { ... }
+}
+```
+
+In this example (if the admin prefix remains the default "Admin") you can reach the Index action at `~/Admin/Person` (or by route name `Person`), because its own action-level attribute took precedence. You can reach Create at `~/Admin/Person/Create` (route name `PersonCreate`) and Edit for the person whose identifier string is "john-doe" at `~/Admin/Person/john-doe` (route name `PersonEdit`).
```

Comment:
```suggestion
In this example, (if the admin prefix remains the default "Admin") you can reach the Index action at `~/Admin/Person` (or by the route name `Person`), because its own action-level attribute took precedence. You can reach Create at `~/Admin/Person/Create` (route name `PersonCreate`) and Edit for the person whose identifier string is "john-doe" at `~/Admin/Person/john-doe` (route name `PersonEdit`).
```